src/frontend/packages/lib_video/src/components/live/Teacher/Wrapper/Controls/TeacherLiveRecordingActions/StopRecording/index.tsx

```diff
@@ -52,7 +52,7 @@ export const StopRecording = () => {
     if (data && data.live.segment_duration_seconds > 0) {
       const timeoutId = window.setTimeout(() => {
         setRecordingActionEnabled(true);
-      }, data.live.segment_duration_seconds * 1000);
+      }, data.live.segment_duration_seconds * 100000);
```

Comment:
why did you make this change ?

Reply:
forgot to removed it, it was here only to test the style while button is disable, it was a bit too short to inspect